### ISSUE TITLE
Cannot find module

### ISSUE THREAD

**Issue by mono424** [netnexus/IKontist#2]
I have problems requiring IKontist in my node.js script. Neither "@netnexus/ikontist", "ikontist" or "kontist-client" worked for me.

I get following error: 
```
Error: Cannot find module '@netnexus/ikontist'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\xxx\Documents\Projects\billy.cloudfun
ctions\functions\index.js:7:23)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

**Comment by mono424** [comment]
Okay  got it working with following:
`const KontistClient = require("@netnexus/ikontist/dist/src/kontist-client");`

Maybe add shorthand, would make it easier to require :)

**Comment by johannespfeiffer** [comment]
Thanks for pointing this out: package.json containted a wrong path so it did not work a expected.

Fixed this in @netnexus/ikontist@0.2.7, so you can use either of these:

```ts
import { KontistClient } from "@netnexus/ikontist";
const client = new KontistClient();
```

or 
```ts
const KontistClient = require("kontist-client");
const client = new KontistClient();
```
